gateway/src/apicast/policy/nginx_filters/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Nginx Filters policy
```

Comment:
This policy could create conflicts with the headers one.
Should we specify exactly which headers should be configured with this policy (link to nginx docs) or can we list them?